packages/vuepress-theme-odyssey/styles/abstracts/_theming.scss

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018-Present Okta, Inc.
```

Comment:
Do we want this to ship Day 1?

How rigorous do you want feedback to be on this theming work?

Reply:
this lives in the theme, and is not part of Odyssey, this is fine to leave in the theme as it provides a starting point for us to do that work.